projects/vdk-core/tests/functional/run/test_run_sql_queries.py

```diff
@@ -75,6 +75,8 @@ def db_connection_recover_operation(self, recovery_cursor: RecoveryCursor) -> No
 
 @mock.patch.dict(os.environ, {})
 def test_run_dbapi_connection_no_such_db_type():
+    logging.getLogger().setLevel(logging.INFO)
```

Comment:
why the flakiness, was the test experiencing intermittent issues on python3.10 only?

Reply:
I was able to reproduce on python 3.8 too. 
I don't think it was related to python 3.10